### DISCUSSION TITLE
Please download the latest image file to be able to update.

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#574] [General]
@rern 
What's the background of the message? My OS is "brand-new" and based of the image of 20210911, not 20210915.
`Linux rAudio 5.10.74-1-raspberrypi4-ARCH #1 SMP PREEMPT Mon Oct 18 11:50:30 MDT 2021 aarch64 GNU/Linux`
What's special with 20210915?

**Comment by klassikmann** [comment]
New post partly nonsense. I've to rethink about the topic.

**Comment by rern** [comment]
Recent updates have cleaned up most of weak points for critical improvements. I'd like to limit them to about 1 month back to narrow down issues. It takes only 10-15 minutes to download-flash-restore a new image while it's a lot more to solve an issue.

You may force update by long-press Update to the latest release. It could be fine or it might break something. I don't know.

**Reply by cmh714** [reply]
long press worked fine for me.....YMMV

**Reply by praveensg8** [reply]
After update to 20211020, While taking settings , Iam getting follwoing errors:
Errors: Unexpected token ] in JSON at position 1638
{ "page" : "system" , "cpuload" : "3.37 1.74 0.75" , "cputemp" : 62.3 , "startup" : "7s (kernel) + 39s (userspace) " , "throttled" : "0x80008" , "time" : "06:13:44 2021-10-21" , "timezone" : "Asia/Kolkata" , "uptime" : "12m" , "uptimesince" : "2021-10-21 06:01" , "audioaplayname" : "allo-katana-dac-audio" , "audiooutput" : "Allo Katana DAC" , "bluetooth" : true , "bluetoothconf" : [ true, false ] , "firmware" : "20211004-1" , "hostapd" : false, "hostname" : "rAudioKATANA" , "kernel" : "5.10.74-1-raspberrypi4-ARCH aarch64" , "lcd" : false, "lcdchar" : false , "lcdcharaddr" : "0x27 0x3F" , "lcdcharconf" : [ 20,"A00","i2c","0x27","PCF8574",15,18,16,21,22,23,24,false ] , "list" : [ {"icon":"microsd","mountpoint":"/","mounted":true,"source":"/dev/mmcblk0p2","size":"2.9GB/30GB"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/?","mounted":false,"source":"/dev/sda1"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/ADATA HD710 PRO","mounted":true,"source":"/dev/sda2","size":"1.5TB/1.9TB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/ALLMUSIC","mounted":false,"source":"//192.168.1.80/ALLMUSIC"} ] , "lcdmodel" : "tft35a" , "mpdoled" : false , "mpdoledconf" : 6 , "online" : true , "ntp" : "pool.ntp.org" , "powerbutton" : false, "powerbuttonconf" : [ 5,40,5 ] , "relays" : false , "rpimodel" : "Raspberry Pi 3 Model B Plus Rev 1.3" , "soc" : "BCM2837B0" , "soccpu" : "4" , "socram" : "952MiB" , "socspeed" : "1400" , "soundprofile" : false , "soundprofileconf" : [ 18000000 ,60 ,1500 ,1000 ] , "version" : "1" , "versionui" : 20211020 , "vuled" : false , "vuledconf" : [ 14,15,18,23,24,25,8 ] , "wlan" : false, "wlanconf" : [ "IN" , ▮] , "wlanconnected" : false}

**Reply by rern** [reply]
Try:
Addons > rAudio 1 > long-press `Update` > ok for `UPDATE` branch
